src/apps/experimental/features/details/hooks/api/useGetItemByType.ts

```diff
@@ -0,0 +1,61 @@
+import type { AxiosRequestConfig } from 'axios';
+import { getUserLibraryApi } from '@jellyfin/sdk/lib/utils/api/user-library-api';
+import { getLiveTvApi } from '@jellyfin/sdk/lib/utils/api/live-tv-api';
+import { useQuery } from '@tanstack/react-query';
+import { type JellyfinApiContext, useApi } from 'hooks/useApi';
+import type { ItemDto } from 'types/base/models/item-dto';
+import { ItemKind } from 'types/base/models/item-kind';
+
+const getItemByType = async (
+    apiContext: JellyfinApiContext,
+    itemType: ItemKind,
+    itemId: string,
+    options?: AxiosRequestConfig
+) => {
+    const { api, user } = apiContext;
+    if (!api) throw new Error('No API instance available');
+    if (!user?.Id) throw new Error('No User ID provided');
```

Comment:
I think the "not null" test (removed) could work for an empty string as well.

Reply:
react query will handle it 
https://github.com/jellyfin/jellyfin-web/blob/b378c46ab56788d83751a8ac6200fa4e4686a132/src/apps/experimental/features/details/hooks/api/useGetItemByType.ts#L59